assets/templates/kubernetes.container/gotext.tpl

```diff
@@ -0,0 +1,17 @@
e94:b3a3", "10.232.83.1", "fe80::bc07:e2ff:fea4:ae01", "10.232.83.1", "fe80::1433:3bff:fe53:5b66", "10.232.83.1", "fe80::a89b:77ff:feae:bcc6", "10.232.83.1", "fe80::24a3:40ff:fe0d:6ba0", "10.232.83.1", "fe80::e050:4eff:fe47:58e2", "10.232.83.1", "fe80::28d6:97ff:fe96:5615", "10.232.83.1", "fe80::88b:57ff:fee3:bc1e", "10.232.83.1", "fe80::184d:f8ff:fe96:4109", "10.232.83.1", "fe80::c898:ebff:fe50:a559", "10.232.83.1", "fe80::8400:edff:fea3:e1ad", "10.232.83.1", "fe80::34f7:7aff:fe03:de89", "10.232.83.1", "fe80::829:49ff:fee5:3ea5", "10.232.83.1", "fe80::9cc1:ccff:fe21:8cd2", "10.232.83.1", "fe80::94cf:87ff:fe1f:7a92", "10.232.83.1", "fe80::5d:14ff:fec1:2d3a" ], "name": "{{ $agentName }}", "id": "{{ $uId }}", "mac":[ "02-42-1F-C0-F0-D2", "02-5D-14-C1-2D-3A", "02-A1-C8-0B-5E-DF", "02-A3-48-EC-21-41", "06-2C-C2-0E-FF-4B", "0A-29-49-E5-3E-A5", "0A-58-08-E1-A6-43", "0A-8B-57-E3-BC-1E", "0A-A9-12-D9-16-0C", "12-42-2A-63-9A-7D", "12-73-5F-4E-81-F7", "12-95-69-35-77-B6", "12-B1-FB-0D-4C-39", "16-0F-01-BE-AA-A0", "16-33-3B-53-5B-66", "1A-4D-F8-96-41-09", "1A-9F-8E-DF-A0-11", "1A-A2-5B-FD-4C-7F", "1A-AE-21-FD-E8-40", "1E-46-56-D5-E3-14", "1E-9C-A1-09-2A-1D", "22-08-D8-E0-20-53", "26-A3-40-0D-6B-A0", "2A-91-AA-D5-A3-3A", "2A-D6-97-96-56-15", "32-32-C7-8C-54-11", "32-72-88-3F-BF-92", "36-44-00-A4-62-67", "36-F7-7A-03-DE-89", "42-01-0A-0A-00-BA", "46-64-F5-60-36-54", "46-F1-C8-E1-3F-37", "4A-D9-98-0D-08-52", "4E-99-7B-9D-7A-00", "52-0B-3A-6B-87-17", "52-84-B9-DF-39-0D", "52-D7-0C-81-5F-58", "56-A0-B4-C4-F5-D5", "56-D2-97-AA-9A-02", "5A-A1-50-DA-6E-C5", "5E-58-24-8F-10-1A", "62-6F-DC-9B-B9-A5", "62-79-1C-48-4A-7A", "6A-24-5A-46-BE-3D", "6A-4B-4E-C9-FA-CE", "72-1E-E8-55-57-C8", "72-52-28-C0-BF-4C", "7A-A0-31-66-6A-A4", "7E-2C-B0-97-4F-E6", "7E-FC-9C-70-78-87", "82-6A-73-D9-0E-AC", "82-B2-07-B9-51-C7", "82-BA-19-5A-58-82", "82-D7-1D-53-29-32", "82-DE-CE-61-02-10", "82-ED-72-F5-A1-33", "86-00-ED-A3-E1-AD", "8A-79-72-4A-60-4A", "8A-E3-4A-82-34-25", "92-A0-37-79-0E-53", "96-BE-33-2C-70-9D", "96-CF-87-1F-7A-92", "96-FA-6D-ED-B3-AB", "9E-95-61-2E-65-35", "9E-C1-CC-21-8C-D2", "A2-37-A4-B9-EB-C7", "A2-53-05-15-D5-C8", "A2-7D-AA-82-43-98", "A6-22-4E-BB-59-76", "A6-3A-53-19-D4-FA", "A6-CC-39-88-CE-C4", "AA-9B-77-AE-BC-C6", "AA-9C-F4-F0-07-64", "AE-11-3C-D5-0D-25", "AE-54-A8-A4-81-1F", "B2-33-C7-E9-AB-FA", "B6-BB-E9-22-E9-C4", "B6-C3-EB-BE-0A-01", "BA-6F-65-B6-5F-2E", "BA-87-AE-57-BF-E7", "BA-8A-04-F8-D8-4E", "BA-AE-1E-A0-3E-10", "BE-07-E2-A4-AE-01", "C2-5B-7B-C3-6B-CB", "C2-8C-52-6A-DF-C4", "C2-94-4E-05-70-5A", "C6-94-B2-D7-B3-4E", "CA-98-EB-50-A5-59", "CE-91-F0-0E-1E-4B", "CE-97-6C-58-40-E2", "D2-DB-58-48-AF-07", "D6-5F-99-9F-D6-D8", "D6-C7-6A-AB-24-95", "DA-06-9A-33-79-D7", "DA-DA-DC-F5-83-84", "DA-E4-2F-50-6E-E0", "E2-35-F1-C8-81-A0", "E2-50-4E-47-58-E2", "E2-6B-7C-F4-90-35", "E6-DE-78-5C-0A-AA", "EE-EC-71-F6-7F-16", "F2-45-C9-18-35-EA", "F2-9E-A0-1D-FA-66", "F6-06-54-94-B3-A3", "F6-AF-C9-EC-1A-8A", "FE-6A-D4-63-97-8D", "FE-F1-90-50-28-AD" ], "architecture":"x86_64"}}}
```

Comment:
```
"name":"{{ generate "container.name"}}", ..., "name":"{{ generate "container.name" }}"
```

beware that this will result in generating two different values from the `container.name` `enum`. so you will end up with a different `container.name` and `kubernetes.container.name`, not sure if that's what you want.

in general, every value that must be the same, every time is "printed" in a single event should generated only once

Reply:
Changed as well. You are right better like this although not affecting our tests